.env.example

```diff
@@ -18,3 +18,8 @@ export SIDEKIQ_PASSWORD=foo
 export MENDELEY_CONSUMER_KEY=foo
 export ERRBIT_API_KEY=foo
 export ERRBIT_HOST=localhost
+
+export SURVEY_EMAIL_USER=foobar
+export SURVEY_EMAIL_PASSWORD=foobar
+export SURVEY_EMAIL_ADDRESS=foobar
+export SURVEY_EMAIL_PORT=foobar
```

Comment:
But aren't we then sending our email through our own relay instead of through the gmail smtp? The one that is having trouble with being blocked by gmail right now? 😉